winui/Ribbon/Getting-Started.md

```diff
@@ -382,6 +382,74 @@ this.rootGrid.Children.Add(sfRibbon);
 
 ![Add ribbon split button in group](Getting-Started-images/add-ribbon-split-button.png)
 
+## Add ribbon toggle button
+
+The [RibbonToggleButton] represents a toggle button, it allows users to switch between two incompatible options and can be added to a ribbon group with different sizes using the `AllowedSizeModes` property. Use the `Content` and `Icon` properties to set the label and icon to the ribbon toggle button.
```

Comment:
@YuvarajMasilamani, modify the content as discussed.

Reply:
Modified